### ISSUE TITLE
Bootstrapping using REPLICATION_HOST instead of HOST

### ISSUE THREAD

**Issue by alexclifford** [zendesk/maxwell#1262]
Running `v1.22.1` I can now attempt a bootstrap when HOST and REPLICATION_HOST are different values. In our scenario the REPLICATION_HOST is a read-only AWS RDS read replica. I believe the SynchronousBootstrapper actions should be taken against HOST instead of REPLICATION_HOST in this scenario?

```
01:36:42,921 DEBUG BootstrapController - starting bootstrap task: #1 database1.entities
01:36:42,922 DEBUG SynchronousBootstrapper - bootstrapping requested for #1 database1.entities
01:36:42,941 DEBUG MysqlSavedSchema - Capturing schemas...
01:36:42,942 DEBUG MysqlSavedSchema - Starting schema capture of 1 databases...
01:36:42,942 DEBUG MysqlSavedSchema - 1/1 Capturing database1...
01:36:42,969 DEBUG MysqlSavedSchema - 1 database schemas captured!
01:36:42,969 DEBUG MysqlSavedSchema - Capturing Default Charset
01:36:42,971 INFO  SynchronousBootstrapper - bootstrapping started for database1.entities
java.sql.SQLException: The MySQL server is running with the --read-only option so it cannot execute this statement
        at com.mysql.jdbc.SQLError.createSQLException(SQLError.java:965)
        at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3978)
        at com.mysql.jdbc.MysqlIO.checkErrorPacket(MysqlIO.java:3914)
        at com.mysql.jdbc.MysqlIO.sendCommand(MysqlIO.java:2530)
        at com.mysql.jdbc.MysqlIO.sqlQueryDirect(MysqlIO.java:2683)
        at com.mysql.jdbc.ConnectionImpl.execSQL(ConnectionImpl.java:2495)
        at com.mysql.jdbc.PreparedStatement.executeInternal(PreparedStatement.java:1903)
        at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2124)
        at com.mysql.jdbc.PreparedStatement.executeUpdateInternal(PreparedStatement.java:2058)
        at com.mysql.jdbc.PreparedStatement.executeLargeUpdate(PreparedStatement.java:5158)
        at com.mysql.jdbc.PreparedStatement.executeUpdate(PreparedStatement.java:2043)
        at snaq.db.CachedPreparedStatement.executeUpdate(CachedPreparedStatement.java:151)
        at com.zendesk.maxwell.bootstrap.SynchronousBootstrapper.setBootstrapRowToStarted(SynchronousBootstrapper.java:176)
        at com.zendesk.maxwell.bootstrap.SynchronousBootstrapper.performBootstrap(SynchronousBootstrapper.java:63)
        at com.zendesk.maxwell.bootstrap.SynchronousBootstrapper.startBootstrap(SynchronousBootstrapper.java:39)
        at com.zendesk.maxwell.bootstrap.BootstrapController.work(BootstrapController.java:65)
        at com.zendesk.maxwell.util.RunLoopProcess.runLoop(RunLoopProcess.java:34)
        at com.zendesk.maxwell.MaxwellContext.lambda$getBootstrapController$0(MaxwellContext.java:392)
        at java.lang.Thread.run(Thread.java:748)
```

**Comment by osheroff** [comment]
Argh yes you're correct.  I need to write a damn test for this scenario.

**Comment by pjebs** [comment]
https://github.com/zendesk/maxwell/pull/1266 ?

**Comment by osheroff** [comment]
fixed in 1.22.2
